crafters/audio/AudioMonophoner/README.md

```diff
@@ -1,3 +1,35 @@
 # AudioMonophoner
 
-makes the audio signal monophonic on doc-level. 
\ No newline at end of file
+**AudioMonophoner** makes the audio signal monophonic on doc-level.
+
+The supported sound formats are:
+WAV, MP3, OGG, AU,FLAC, RAW, AIFF, AIFF-C, PAF, SVX, NIST, VOC, IRCAM, W64, MAT4, MAT5, PVF, XI, HTK, SDS, AVR, WAVEX, SD2, CAF, WVE, MPC2K, RF64.
+
+## Usage
+
+The **AudioMonophoner** executor uses the [`librosa`](https://github.com/librosa/librosa)  library to convert the input `blob` (`ndarray` representation of the audio signal) to a dictionary of `monophonic` audio signal. 
+
+We use Pod images in several ways:
+
+1. Run with Docker: `docker run`
+   ```bash
+    docker run --rm -p 55555:55555 -p 55556:55556 jinahub/pod.crafter.audiomonophoner:0.0.9-1.0.1 --port-in 55555 --port-out 55556
+    ```
+    
+2. Run with Flow API
+   ```python
+    from jina.flow import Flow
+
+    f = (Flow()
+        .add(name='my_encoder', uses='docker://jinahub/pod.crafter.audiomonophoner:0.0.9-1.0.1', port_in=55555, port_out=55556))
+    ```
+    
+3. Run with Jina CLI
+   ```bash
+    jina pod --uses docker://jinahub/pod.crafter.audiomonophoner:0.0.9-1.0.1 --port-in 55555 --port-out 55556
+    ```
+    
+4. Run with conventional local usage. Needs `uses` argument:
+   ```bash
+    jina pod --uses hub/example/audiomonophoner.yml --port-in 55555 --port-out 55556
```

Comment:
`hub/example/audiomonophoner.yml` - Let's change it to the correct relative path - `crafters/audio/AudioMonophoner/audiomonophoner.yml`